Packs/GCP-Enrichment-Remediation/Scripts/GCPOffendingFirewallRule/README.md

```diff
@@ -0,0 +1,42 @@
+Determine potential offending firewall rules in GCP based on port, protocol and possibly target tags (network tags).
```

Comment:
It's not necessary,
but you can include a script example in the readme file by using
`demisto-sdk generate-docs -i Packs/GCP-Enrichment-Remediation/Scripts/GCPOffendingFirewallRule/GCPOffendingFirewallRule.yml -e '!GCPOffendingFirewallRule port=test protocol=test network_url=test' -ngr`.
For more information, please visit https://xsoar.pan.dev/docs/documentation/readme_file#creating-documentation.
See the example in the `Packs/DemistoRESTAPI/Scripts/GetIncidentTasks/README.md` file

Reply:
i could do this on XSOAR but we mostly develop for XSIAM/Xpanse